js/angular/app/index.html

```diff
@@ -79,6 +80,7 @@ <h1 class="application-title">Open Transit Indicators</h1>
     <script src="bower_components/leaflet-utfgrid/dist/leaflet.utfgrid.js"></script>
     <script src="bower_components/leaflet-label/dist/leaflet.label.js"></script>
     <script src="bower_components/leaflet-tilelayer-geojson/TileLayer.GeoJSON.js"></script>
+    <script src="bower_components/leaflet.draw/dist/leaflet.draw-src.js"></script>
```

Comment:
Apparently a library update added the src file.  Didn't exclude that one, as it's benign.